model/src/main/java/org/cbioportal/genome_nexus/model/TranscriptConsequence.java

```diff
@@ -63,6 +63,7 @@
     private Double siftScore;
     private String siftPrediction;
     private String exon;
+    private String uniprotId;
```

Comment:
Not sure if we want to have uniprot id in transcript consequence, because I saw we label transcript consequence with `@Deprecated` https://github.com/genome-nexus/genome-nexus/blob/master/model/src/main/java/org/cbioportal/genome_nexus/model/VariantAnnotationSummary.java#L17`

Reply:
I think it is okay to have `uniprotId` here. Seems like we only deprecate a specific old list here, not the actual model.